src/main/java/frc/team555/robot/AutoModes.java

```diff
@@ -0,0 +1,18 @@
+package frc.team555.robot;
+
+import edu.wpi.first.wpilibj.smartdashboard.SmartDashboard;
+import org.montclairrobotics.sprocket.auto.AutoMode;
+import org.montclairrobotics.sprocket.states.State;
+
+import java.util.ArrayList;
+
+public class AutoModes {
+    public static SmartDashboard dashboard;
+
+    public static AutoMode buildDashboardAuto(){
```

Comment:
What's this method supposed to do? What's the code in here supposed to accomplish?

Reply:
Oh sorry, I didn't mean to have that, it was an accident